web/app.php

```diff
@@ -67,7 +67,11 @@
 // If you are behind one or more trusted reverse proxies, you might want to set them in SYMFONY_TRUSTED_PROXIES environment
 // variable in order to get correct client IP
 if ($trustedProxies = getenv('SYMFONY_TRUSTED_PROXIES')) {
-    Request::setTrustedProxies(explode(',', $trustedProxies));
+    if ($trustedProxies === 'TRUST_REMOTE') {
+        Request::setTrustedProxies(array($request->server->get('REMOTE_ADDR')));
```

Comment:
```suggestion
        Request::setTrustedProxies([$request->server->get('REMOTE_ADDR')]);
```
Nitpick: Old array syntax